doc/articles/toc.yml

```diff
@@ -41,16 +41,16 @@
 
 - name: Samples & Tutorials
   items:
-    - name: Tutorial 1
-      href: getting-started-tutorial-1.md
-    - name: Tutorial 2
-      href: getting-started-tutorial-2.md
-    - name: Tube Player
-      topicHref: external/workshops/tube-player/README.md
-      href: external/workshops/tube-player/toc.yml
+    # - name: Tutorial 1
+    #   href: getting-started-tutorial-1.md
```

Comment:
let's remove entirely (files included)

Reply:
yep was going to leave for separate PR but yeh easy enough